cpp/QuickSort.cpp

```diff
@@ -0,0 +1,68 @@
+#include <iostream>
+ 
+using namespace std;
+ 
+void quick_sort(int[],int,int);
+int partition(int[],int,int);
+ 
+int main()
+{
+    int a[50],n,i;
+    cout<<"How many elements?";
```

Comment:
I advice generally not to submit files with main functions, but just the library functions. For example, this change could be good with just a function called quicksort()